lib/sle2docker/cli.rb

```diff
@@ -50,6 +50,10 @@ def version
 
     private
 
+    def ensure_can_access_dockerd
+      Docker.info
```

Comment:
This will cause an exception if the docker daemon in unreachable. Are we handling that in a nice way?

Reply:
The output is exactly the same as it was before, a text error from docker client not being capable to find the unix socket. I tried to include an exception management with a `begin` and `rescue` clause, but rubocop complaint about being redundant in case no specific Exception class was matched. Since I did not want to limit the clause to some specific exception class (mainly because Docker class is not part of our code) I just opted to leave it as is and let Docker gem fail.

Reply:
👍